crates/nu-command/src/math/mode.rs

```diff
@@ -130,11 +130,11 @@ pub fn mode(values: &[Value], _span: Span, head: Span) -> Result<Value, ShellErr
                 NumberTypes::Filesize,
             )),
             Value::Error { error, .. } => Err(*error.clone()),
-            other => Err(ShellError::UnsupportedInput {
+            _ => Err(ShellError::UnsupportedInput {
                 msg: "Unable to give a result with this input".to_string(),
                 input: "value originates from here".into(),
                 msg_span: head,
-                input_span: other.span(),
+                input_span: span,
```

Comment:
While debugging:

Before this change
![image](https://github.com/user-attachments/assets/0ad95125-814a-4d55-9821-e245849a70e6)

After this change
![image](https://github.com/user-attachments/assets/ac137524-44fe-48aa-992f-91c0d7f72659)

But in the end I removed the possibility for this command to accept ranges, so you want see it. Now:

![image](https://github.com/user-attachments/assets/5d9a2e32-7116-439f-a145-36d49ff113a4)

Reply:
Yeah the span change looks correct, and the decision to omit this for `math mode` seems fine to me. Pretty pointless to do on a `range`.